### ISSUE TITLE
rationale for default values of constants in exponentiated gradient

### ISSUE THREAD

**Issue by romanlutz** [fairlearn/fairlearn#96]
Would be awesome to have a description of where these numbers come from or why they might be reasonable.

**Comment by TamaraAtanasoska** [comment]
any pointers @MiroDudik for where to retrieve this information from?

**Comment by romanlutz** [comment]
I think this is only in @MiroDudik's memory 😆 or perhaps the paper has some mention of it. Fairlearn started as the repo for that algorithm, so if it's not already in the code then @MiroDudik is our only hope 😆

**Comment by MiroDudik** [comment]
@romanlutz which specific numbers does this refer to?

I'm guessing it might be:
* `eta0` (default learning rate parameter)
* `max_iter` (default max number of iterations)
* internal (invisible to user) explicit optimization parameters from [_exponentiated_gradient/_constants.py](https://github.com/fairlearn/fairlearn/blob/v0.12.0/fairlearn/reductions/_exponentiated_gradient/_constants.py):
   * `_ACCURACY_MUL`, ... , `_MIN_ITER`

The honest answer is that all of these were tuned to perform well across the experiments reported in [Agarwal et al. 2018](https://proceedings.mlr.press/v80/agarwal18a.html). So, we expect them to work well on qualitatively similar tasks.

We did not publish any notebooks for the experiments in that paper. We could discuss whether this is something that we think would be valuable to add to the repository and we could open this as an issue.

**Comment by romanlutz** [comment]
Yes, those ones! I think this description is already pretty much what we were looking for at that time. It's sometimes hard to say whether people put thought into constants unless it's mentioned how they came up with them. I figured there must be some rationale 🙂

**Comment by TamaraAtanasoska** [comment]
I am not sure if we need to do more work about this, except possibly adding an optional sentence that these were constants that came from the initial experimentation. I think most people who've done similar work will know this implicitly though :) I would lean towards closing this unless you feel strongly about documenting this further @romanlutz
